src/app/shared/models/v2/cup/cup-rating-position-search.model.ts

```diff
@@ -0,0 +1,4 @@
+export class CupRatingPositionSearch {
```

Comment:
there is a separate folder for search-clases: `src/app/shared/models/search/cup`